www/js/controllers/menu_controller.js

```diff
@@ -26,4 +26,10 @@ app.controller('MenuCtrl', function($scope, $state, $ionicLoading, BarcodeServic
   $scope.redirectHome = function() {
     $state.go("app.home");
   };
+
+  $scope.changeCompany = function() {
+    var seller = UserService.currentUser();
+    seller.removeDevice();
```

Comment:
There should be a confirmation dialog upon changing company as stated in the issue. See this mockup: https://moqups.com/linesarefuzzy/cxlIqKWm/p:a1ede49f4